test/gitlab_merge_request_test.go

```diff
@@ -97,6 +99,32 @@ func TestGitlabMergeRequest(t *testing.T) {
 	for i := 0; i < len(prsNew.Items); i++ {
 		assert.Equal(t, "Merge Request", prsNew.Items[i].Annotations[keys.EventType])
 	}
+
+	runcnx.Clients.Log.Infof("Sending /retest comment on MergeRequest %s/-/merge_requests/%d", projectinfo.WebURL, mrID)
+	_, _, err = glprovider.Client.Notes.CreateMergeRequestNote(opts.ProjectID, mrID, &clientGitlab.CreateMergeRequestNoteOptions{
+		Body: clientGitlab.Ptr("/retest"),
+	})
+	assert.NilError(t, err)
+	sopt = wait.SuccessOpt{
+		Title:           commitTitle,
+		OnEvent:         "Note",
+		TargetNS:        targetNS,
+		NumberofPRMatch: 5, // this is the max we get in repos status
```

Comment:
It should be 6 i believe :thinking: 

Because before `/retest`  there were 4 PRs and after adding  `/retest` 2 more PRs will be triggered so total 6 ?

Reply:
If the value of 5 is incorrect, shouldn't that test fail or block on waiting?

Reply:
i explained in the comment, we have a max of 5 on repo status.....

Reply:
probably should be renamed NumberOfRepoStatus tho but 🤷🏻

Reply:
You read my thought :-)